pkg/workloads/containerd/watcher.go

```diff
@@ -25,7 +25,7 @@ import (
 	"time"
 
 	"github.com/cilium/cilium/common/addressing"
-	"github.com/cilium/cilium/pkg/endpoint"
+	endpointid "github.com/cilium/cilium/pkg/endpoint/id"
```

Comment:
Merge this commit with the id package commit in order not to break git bisecting as the previous commit alone will not compile.